our strftime doesn't error out with %-d but just outputs -d rather
than doing whatever this test wants

Index: dateutil/test/test_parser.py
--- dateutil/test/test_parser.py.orig
+++ dateutil/test/test_parser.py
@@ -104,6 +104,8 @@ PARSER_TEST_CASES = [
 # Check that we don't have any duplicates
 assert len(set([x[0] for x in PARSER_TEST_CASES])) == len(PARSER_TEST_CASES)
 
+if sys.platform.startswith('openbsd'):
+    PLATFORM_HAS_DASH_D = False
 
 @pytest.mark.parametrize("parsable_text,expected_datetime,assertion_message", PARSER_TEST_CASES)
 def test_parser(parsable_text, expected_datetime, assertion_message):
